progenitor-impl/src/method.rs

```diff
@@ -902,6 +919,16 @@ impl Generator {
                     // returns an error in the case of a serialization failure.
                     .form_urlencoded(&body)?
                 }),
+                (
+                    OperationParameterKind::Body(
+                        BodyContentType::FormData
+                    ),
+                    OperationParameterType::Form(_),
+                ) => {
+                    Some(quote! {
+                    // This uses progenitor_client::RequestBuilderExt which sets up a simple form data based on bytes
+                    .form_from_raw(body.as_form())?
```

Comment:
did you think about unrolled the `as_form` logic here? That seems potentially cleaner than an `impl` on the generated type.